src/span-data.js

```diff
@@ -92,7 +97,10 @@ SpanData.prototype.createChildSpanData = function(name, skipFrames) {
 };
 
 SpanData.prototype.addLabel = function(key, value) {
-  this.span.setLabel(key, value);
+  key = traceUtil.truncate(key, constants.TRACE_SERVICE_LABEL_KEY_LIMIT);
+  var val = typeof value === 'object' ? util.inspect(value) : '' + value;
```

Comment:
nit: This seems assymetric. You assign to the argument `key` but keep create an additional names for `value`. Keep local variables for both or neither.